spec/std/enumerable_spec.cr

```diff
@@ -37,6 +37,26 @@ describe "Enumerable" do
     end
   end
 
+  describe "any? with block" do
+    it "returns true" do
```

Comment:
I'd extend on the description a bit to be more descriptive of what the described method does - e.g. "it returns true if one element fulfills the condition" (applies to all specs)